### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by Ellenn-A**
# Update to documentation

- please feel free to correct any syntactic mistakes ..or any other mistakes

### CODE REVIEW

**Review by msterle — COMMENTED** [review]
lftm

File: README_VERITABLE.md [hunk 1]
```diff
@@ -34,12 +34,28 @@ The Aries Framework JavaScript REST API is the most convenient way for self-sove
...
+
 > The OpenAPI spec is generated from the model classes used by Aries Framework JavaScript. Due to limitations in the inspection of these classes, the generated schema does not always exactly match the expected format. Keep this in mind when using this package. If you encounter any issues, feel free to open an issue.
 
+### Rest Client Args
+
+The CLI args are defined under `src > cli.ts ` They are used to start up a service.
+| Argument added | Required | Default | Description |
+| :------------- | :------: | :------ | :------- |
+| ipfs-origin | Y | http://ipfs0:5001 | IPFS endpoint |
+|opa-origin |Y |http://opa:8181 |OPA endpoint |
```
Comment (jonmattgray) [inline README_VERITABLE.md:55]: I've realised this is missing in `cli.ts` could you please add it as an option?
```
  .option('opa-origin', {
    string: true,
    default: 'http://localhost:8181'
  })
```

File: README_VERITABLE.md [hunk 2]
```diff
@@ -86,6 +102,8 @@ Network name: `testnet`
 
 ### Via CLI
 
+<a id="via-cli"></a>
+
```
Comment (jonmattgray) [inline README_VERITABLE.md:106]: remove?

File: README_VERITABLE.md (hunk 1 above)
Comment (Ellenn-A) [reply README_VERITABLE.md:55]: Ty @jonmattgray, it's been added, so should I change the opa-origin default to `http://localhost:8181` ? and does that mean that it is not required? 
Equally should I change the ipfs-origin not to have a default?

Comment (jonmattgray) [reply README_VERITABLE.md:55]: talked in standup

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**